app/src/main/java/com/nutomic/syncthingandroid/views/FoldersAdapter.java

```diff
@@ -82,18 +82,23 @@ public View getView(int position, View convertView, @NonNull ViewGroup parent) {
                 binding.state.setText(getContext().getString(R.string.status_outofsync));
                 binding.state.setTextColor(ContextCompat.getColor(getContext(), R.color.text_red));
             } else {
-                binding.state.setText(getLocalizedState(getContext(), model.state, percentage));
-                switch(model.state) {
-                    case "idle":
-                        binding.state.setTextColor(ContextCompat.getColor(getContext(), R.color.text_green));
-                        break;
-                    case "scanning":
-                    case "cleaning":
-                    case "syncing":
-                        binding.state.setTextColor(ContextCompat.getColor(getContext(), R.color.text_blue));
-                        break;
-                    default:
-                        binding.state.setTextColor(ContextCompat.getColor(getContext(), R.color.text_red));
+                if (folder.paused) {
+                    binding.state.setText(getContext().getString(R.string.state_paused));
+                    binding.state.setTextColor(ContextCompat.getColor(getContext(), R.color.text_black));
+                } else {
+                    binding.state.setText(getLocalizedState(getContext(), model.state, percentage));
+                    switch(model.state) {
+                        case "idle":
+                            binding.state.setTextColor(ContextCompat.getColor(getContext(), R.color.text_green));
+                            break;
+                        case "scanning":
+                        case "cleaning":
```

Comment:
This state does not exist, just remove it.

Reply:
I think you meant the "cleaning" state above as mentioned here? I removed it. Never saw it in syncthing before.